controllers/controllers/services/cfservicebinding_controller_test.go

```diff
@@ -318,6 +335,47 @@ var _ = Describe("CFServiceBinding.Reconcile", func() {
 				})))
 			})
 		})
+		When("the cfapp vcap services secret status is not set", func() {
+			BeforeEach(func() {
+				cfAppStatus = korifiv1alpha1.CFAppStatus{}
+			})
+			It("requeues the request", func() {
+				Expect(reconcileResult).To(Equal(ctrl.Result{RequeueAfter: 2 * time.Second}))
+				Expect(reconcileErr).NotTo(HaveOccurred())
+
+				Expect(fakeStatusWriter.UpdateCallCount()).To(Equal(2))
+				_, serviceBindingObj, _ := fakeStatusWriter.UpdateArgsForCall(0)
+				updatedCFServiceBinding, ok := serviceBindingObj.(*korifiv1alpha1.CFServiceBinding)
+				Expect(ok).To(BeTrue())
+				Expect(updatedCFServiceBinding.Status.Binding.Name).To(BeEmpty())
+				Expect(updatedCFServiceBinding.Status.Conditions).To(ContainElement(MatchFields(IgnoreExtras, Fields{
+					"Type":   Equal("VCAPServicesSecretAvailable"),
+					"Status": Equal(metav1.ConditionFalse),
+				})))
+			})
+		})
+		When("the vcap services secret isn't found", func() {
+			BeforeEach(func() {
+				fakeClient.GetReturnsOnCall(4, apierrors.NewNotFound(schema.GroupResource{}, cfApp.Status.VCAPServicesSecretName))
+			})
+
+			// Do we actually want to requeue this request? A missing secret is a system consistency error that we
```

Comment:
I would suggest addressing those questions in a chore/story rather than in a comment in tests

Reply:
Should now be covered by https://github.com/cloudfoundry/korifi/issues/1237
Removed.